pages/sunos/devfsadm.md

```diff
@@ -0,0 +1,15 @@
+# devfsadm
+
+> Administration command for /dev. Maintains the /dev namespace.
```

Comment:
I'd say format "/dev" using backticks, to highlight that it is a path. @igorshubovych any thoughts?